threedi_modelchecker/simulation_templates/laterals/dwf_calculator.py

```diff
@@ -0,0 +1,127 @@
+from threedi_api_client.openapi.models.lateral import Lateral
+from threedi_modelchecker.simulation_templates.utils import (
+    strip_dict_none_values,
+)
+from typing import List
+from typing import Union
+
+import sqlite3
+
+
+# Default values
+DWF_FACTORS = [
+    [0, 0.03],
+    [1, 0.015],
+    [2, 0.01],
+    [3, 0.01],
+    [4, 0.005],
+    [5, 0.005],
+    [6, 0.025],
+    [7, 0.080],
+    [8, 0.075],
+    [9, 0.06],
+    [10, 0.055],
+    [11, 0.05],
+    [12, 0.045],
+    [13, 0.04],
+    [14, 0.04],
+    [15, 0.035],
+    [16, 0.035],
+    [17, 0.04],
+    [18, 0.055],
+    [19, 0.08],
+    [20, 0.07],
+    [21, 0.055],
+    [22, 0.045],
+    [23, 0.04],
+    [24, 0.0],  # Timeseries for laterals should contain 25 values
+]
+
+
+def read_dwf_per_node(spatialite_path: str) -> List[Union[int, float]]:
+    """Obtains the DWF per connection node per second a 3Di model sqlite-file."""
+
+    conn = sqlite3.connect(spatialite_path)
+    cursor = conn.cursor()
+
+    # Create empty list that holds total 24h dry weather flow per node
+    dwf_per_node_per_second = []
+
+    # Create a table that contains nr_of_inhabitants per connection_node and iterate over it
+    for row in cursor.execute(
```

Comment:
Would be nice to use 'condenser' package here, this allows fast SQLAlchemy to Numpy conversion.

Reply:
Skipped for now, made a ticket so it can be picked up later: https://github.com/nens/threedi-modelchecker/issues/145